internal/fpath/path.go

```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2018 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package fpath
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+// FPath is an OS independently path handling structure
+type FPath struct {
+	local  bool   // if file is local
+	scheme string // url scheme
+	bucket string // set if remote scheme
+	path   string // local file path or Storj path (without bucket), cleaned, with forward slashes
+}
+
+//WindowsPrefix is a filter for the Universal Naming Convention path (Windows)
+var WindowsHandlingPrefix = regexp.MustCompile(`^\\\\\?\\(UNC\\)?`)
```

Comment:
These shouldn't be usually public. Also `windowsUNCPrefix`

Reply:
@egonelbre This regex is checking for `\\?\` and `\\?\UNC\`. Thats why i called it handling prefix instead of UNC.